src/fileio.cpp

```diff
@@ -0,0 +1,62 @@
+#include <fstream>
+#include <iostream>
+#include <vector>
+#include "fileio.h"
+
+void import_stl (string filepath) {
+}
+
+void export_stl (World *world, string filepath) {
+  ofstream fs;
+  fs.open(filepath + ".stl");
```

Comment:
`fs.open` creates it if it doesn't exist; the `!fs.is_open()` case would only occur if there was some error opening the file for writing, e.g. if another process is currently writing to it or something.